ddtrace/tracer/tracer_test.go

```diff
@@ -238,6 +238,50 @@ func TestTracerStartSpan(t *testing.T) {
 	})
 }
 
+func TestP0Dropping(t *testing.T) {
+	t.Run("p0 are kept by default", func(t *testing.T) {
```

Comment:
Just one word please. It's what we do everywhere. Just to stay consistent please 🙏🏻